### PR TITLE
Add Bitwarden.CLI 2023.2.0

### PR DESCRIPTION
**Pull request by al-lac**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/98394)

### CODE REVIEW

**Comment by al-lac** [comment]
@JohnMcPMS not sure what the issue is here, it installs fine locally.

**Comment by stephengillie** [comment]
Hi @al-lac,

The executable is expecting arguments. Maybe the argument `--cleanexit` should be added somehow?
```
Could not find dir, "C:\Users\WDAGUtilityAccount\AppData\Roaming\Bitwarden CLI"; creating it instead.
Could not find data file, "C:\Users\WDAGUtilityAccount\AppData\Roaming\Bitwarden CLI\data.json"; creating it instead.
Usage: bw [options] [command]

Options:
  --pretty                                    Format output. JSON is tabbed with two spaces.
  --raw                                       Return raw output instead of a descriptive message.
  --response                                  Return a JSON formatted version of response output.
  --cleanexit                                 Exit with a success exit code (0) unless an error is thrown.
...
```

File: manifests/b/Bitwarden/CLI/2023.2.0/Bitwarden.CLI.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+InstallerType: zip
+ReleaseDate: 2023-02-16
+Installers:
+- Architecture: x64
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: .\bw.exe status --quiet
+    PortableCommandAlias: bw
+  InstallerUrl: https://github.com/bitwarden/clients/releases/download/cli-v2023.2.0/bw-windows-2023.2.0.zip
+  InstallerSha256: b2384e5e871be0f709ddece33dc775bfcbfc95a5d0faaf61f1f4b245245dea0e
```
Comment (stephengillie) [inline manifests/b/Bitwarden/CLI/2023.2.0/Bitwarden.CLI.installer.yaml:15]:
```suggestion
  - RelativeFilePath: .\bw.exe
    PortableCommandAlias: bw
  InstallerUrl: https://github.com/bitwarden/clients/releases/download/cli-v2023.2.0/bw-windows-2023.2.0.zip
  InstallerSha256: b2384e5e871be0f709ddece33dc775bfcbfc95a5d0faaf61f1f4b245245dea0e
  InstallerSwitches:
    Silent: status --quiet
```

**Comment by stephengillie** [comment]
This package is expecting arguments:
```
Usage: bw [options] [command]

Options:
  --pretty                                    Format output. JSON is tabbed with two spaces.
  --raw                                       Return raw output instead of a descriptive message.
  --response                                  Return a JSON formatted version of response output.
  --cleanexit                                 Exit with a success exit code (0) unless an error is thrown.
  --quiet                                     Don't return anything to stdout.
  --nointeraction                             Do not prompt for interactive user input.
  --session <session>                         Pass session key instead of reading from env.
  -v, --version                               output the version number
...
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**